roles/dosomething.base/vars/main.yml

```diff
@@ -1,2 +1,7 @@
 ---
 # vars file for ds-base
+
```

Comment:
It would make more sense to keep it out of default vars: this is DoSomething-specefic.
You can declare this in `group_vars/all/main.yml` to set this variable for all groups.